plugins/hashsum_download/web_client/templates/hashsum_download_fileInfo.jade

```diff
@@ -0,0 +1,6 @@
+if file.has('sha512')
```

Comment:
That would be more elegant. Unfortunately this plugin is taking advantage of a data model that predates it. That data model was not created with consideration for this use case 😄